resources/js/src/app/components/itemList/ItemSearch.js

```diff
@@ -59,13 +59,10 @@ Vue.component("item-search", {
 
     methods:
     {
-        prepareSearch()
+        search()
```

Comment:
Why was this changed?

Reply:
Can't remember exactly. Either prepareSearch() was no longer needed after refactoring or it was actually causing unwanted behavior.

Reply:
It relates to the changes in scripts.twig. There was code in two places that did essentially the same thing. I refactored it and the separate prepareSearch() function was no longer needed.